src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ValueProviderResult.cs

```diff
@@ -38,9 +37,9 @@ public struct ValueProviderResult : IEquatable<ValueProviderResult>, IEnumerable
         /// <summary>
         /// Creates a new <see cref="ValueProviderResult"/> using <see cref="CultureInfo.InvariantCulture"/>.
         /// </summary>
-        /// <param name="value">The submitted value.</param>
-        public ValueProviderResult(string value)
-            : this(value, _invariantCulture)
+        /// <param name="values">The submitted values.</param>
+        public ValueProviderResult(StringValues values)
```

Comment:
We should have a test that `new ValueProviderResult(...)` with a `StringValues` with no values is equal to `ValueProviderResult.None`.